src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Parser.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.DotnetRuntime.Extensions;
+using GeneratorAttributeSyntaxContext = Microsoft.CodeAnalysis.DotnetRuntime.Extensions.GeneratorAttributeSyntaxContext;
```

Comment:
Can you elaborate on why this is needed?

Reply:
@lbussell added in https://github.com/dotnet/installer/pull/14162, there was some conflict in roslyn dependency at the time. i'm testing if it is still needed: https://github.com/dotnet/installer/pull/14545, doesn't seem like it. i'll close this one.